### PR TITLE
Initial commit 'New Structure'

### PR DESCRIPTION
**Pull request by SrOliver202023**
New structure recommendation inspired by rocketeseat's nodejs trail. #2

### CODE REVIEW

File: ormconfig.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+   "type": "postgres",
+   "host": "localhost",
+   "port": 5432,
+   "username": "admin",
+   "password": "@adm1n202234.",
+   "database": "books_db",
```
Comment (d3vlopes) [inline ormconfig.json:7]: Adiciona esse arquivo no `.gitignore` e depois faz uma copia com o nome de `ormconfig-example.json` mais remove as credências do banco

**Review by SrOliver202023 — COMMENTED** [review]
Initial commit, new structure

**Review by d3vlopes — CHANGES_REQUESTED** [review]

**Review by SrOliver202023 — COMMENTED** [review]
example files created and duplicate files removed

**Review by SrOliver202023 — COMMENTED** [review]
Final commit #

**Review by d3vlopes — APPROVED** [review]

**Pull request merged**